### ISSUE TITLE
Blocks: Copy-paste two blocks generates one block only when pasted

### ISSUE THREAD

**Issue by retnonindya** [Automattic/wp-calypso#51532]
A user reported unexpected behavior on copy-pasting blocks:

When selecting texts on multiple blocks and copied it, the result is only one block.
 
### Steps to reproduce the behavior

1. Open/create new post/page with multiple Paragraph blocks or a mix of Heading block and Paragraph block.
2. Select the texts that you want to copy
3. Click CMD + C (or Ctrl + C) to copy
4. See "Copied X blocks to clipboard" -- at this point, the expectation is to have all highlighted/selected contents to be copied and pasted properly
5. Open/create new post/page, or put cursor under the copied texts (on the same post/page), click CMD + V (Ctrl + V) to paste
6. Only one block pasted, the first block -- the following block's contents are not pasted

Also, note when the Heading block is copy-pasted, the output renders as Paragraph block/losing formatting. This has been reported here:
https://github.com/Automattic/wp-calypso/issues/50703 

#### What I expected to happen
The expectation is to have all highlighted/selected contents to be copied and pasted properly

#### What actually happened
Only one block pasted, the first block -- the following block's contents are not pasted

### Context
User report and I able to replicate it

#### Browser / OS version
I'm using Chrome Version 89.0.4389.90. The user is using Firefox 78.0 on Mac OS X 10.10.0.

#### Is this specific to the applied theme? Which one?
All themes. I can confirm this happened on theme Floral (advanced theme) and theme Seedlet (block editor-supporting theme)

#### Does this happen on simple or atomic sites or both?
Both

#### Is there any console output or error text?

Console output looks like this:

```
Received HTML:

<meta charset='utf-8'>Why do we use it?

Received plain text:

 Why do we use it?
```

#### Level of impact (Does it block purchases? Does it affect more than just one site?)
This affects more than one site and not blocking any purchases

#### Screenshot / Video: If applicable, add screenshots to help explain your problem.

Screen recording in GIF format:
https://cldup.com/LMXC6jlQVy.gif

**Comment by Sandkorner** [comment]
I guess a workaround here is, or maybe it is intended as so, to use the copy option from the above toolbar for several blocks at once.

Like so: https://snipboard.io/SxI5DP.jpg

This will show also in the left bottom corner, that 2 blocks (or more if you wish) were copied.

After that you can normally paste (command/control + V).

**Comment by retnonindya** [comment]
Closing this in favor of this report:

https://github.com/WordPress/gutenberg/issues/30490
